test/inductor/test_fused_attention.py

```diff
@@ -171,7 +171,7 @@ def sfdp_pattern_6(query, key, value):
             attn_weight = torch.dropout(attn_weight, 0.5, True)
             return attn_weight @ value
 
-        self._check_common(sfdp_pattern_6, contains=False)
+        self._check_common(sfdp_pattern_6, contains=False, has_fuse_pattern=False)
```

Comment:
NB: `fuse_attention` counter is now 0, which means some `_sfdp_replacement` is not executed. Should I investigate? Or are these supposed to be somewhat flaky?

Reply:
This shouldn't happen, no.